### PR TITLE
docs: visual fixes

### PR DESCRIPTION
**Pull request by mainframev**
Visual fixes acc. to the [task](https://jira.kiwi.com/browse/ORBIT-1836)
 Storybook: https://orbit-silvenon-docs-visual-fixes.surge.sh

### CODE REVIEW

File: docs/src/components/ReactExample/Example.tsx [hunk 1]
```diff
@@ -66,9 +66,14 @@ const Example = ({
 }: Props) => {
   const [isEditorOpened, setOpenEditor] = React.useState(false);
   const [isPlaygroundOpened, setPlaygroundOpened] = React.useState(false);
+  const [bg, setBackground] = React.useState<BgType>("white");
```
Comment (silvenon) [inline docs/src/components/ReactExample/Example.tsx:69]: Use a more descriptive name for `bg` to differentiate it from `background`, like `selectedBackground`.

File: docs/src/components/ReactExample/components/Board.tsx [hunk 2]
```diff
@@ -75,6 +107,37 @@ const Board = ({
...
+                  label="grid"
+                  onChange={handleBackground}
+                />
+              </Stack>
+            }
+          >
+            <ButtonLink type="secondary">
+              <DarkMode />
+            </ButtonLink>
+          </Popover>
```
Comment (silvenon) [inline docs/src/components/ReactExample/components/Board.tsx:140]: Very cool! 😍

**Review by silvenon — CHANGES_REQUESTED** [review]

File: docs/src/components/DocLayout/index.tsx [hunk 3]
```diff
@@ -222,7 +222,7 @@ export default function DocLayout({
               )}
             </StyledMain>
           </StyledMiddle>
-          <Footer hasGradient />
+          <Footer />
```
[the remark this answers is not in the document]
Comment (mainframev) [reply docs/src/components/DocLayout/index.tsx:225]: i don't know, designers decided to remove the gradient on footer

[the remark this answers is not in the document]
Comment (silvenon) [reply docs/src/components/DocLayout/index.tsx:225]: We got our answer on Slack: the left navigation should fade out, but not the main content box.

[the remark this answers is not in the document]
Comment (mainframev) [reply docs/src/components/DocLayout/index.tsx:225]: Yep, saw that ;)

File: docs/src/components/DocLayout/primitives/StyledMobileOutdent.tsx [hunk 4]
```diff
@@ -13,6 +13,7 @@ const StyledMobileOutdent = styled.div`
 
   > *:last-child {
     flex: 1;
+    margin-bottom: 4rem;
```
[the remark this answers is not in the document]
Comment (mainframev) [reply docs/src/components/DocLayout/primitives/StyledMobileOutdent.tsx:16]: yep, shadow will be removed, but margin still has to be there

File: docs/src/components/DocLayout/primitives/StyledMobileOutdent.tsx [hunk 5]
```diff
@@ -13,6 +13,7 @@ const StyledMobileOutdent = styled.div`
 
   > *:last-child {
     flex: 1;
+    margin-bottom: 3rem;
```
Comment (silvenon) [inline docs/src/components/DocLayout/primitives/StyledMobileOutdent.tsx:16]: Just a reminder, I think this gap is unnecessary. My suggestion is to remove the margin.

Screen Shot 2021-09-09 at 19 17 29

**Review by silvenon — APPROVED** [review]
Sold!

**Pull request merged**